qa/config/crimson_bluestore.yaml

```diff
@@ -0,0 +1,24 @@
+overrides:
+  ceph:
+    fs: xfs
+    conf:
+      osd:
+        # crimson's osd objectstore option
+        crimson osd objectstore: bluestore
+        bluestore block size: 96636764160
+        debug bluestore: 20
```

Comment:
Probably this is the better place for
```
debug alienstore: 20
```
?

https://github.com/ceph/ceph/pull/60770

Reply:
Yes, previously we had a common `bluestore.yaml` that I tried to reuse. Thanks!